CHANGELOG.md

```diff
@@ -6,6 +6,16 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
 
+## [1.4.0](https://github.com/stylelint/vscode-stylelint/compare/v1.3.0...v1.4.0) (2024-03-xx)
+
+### Changed
+
+- Bump vscode language server dependencies ([#521](https://github.com/stylelint/vscode-stylelint/pull/521)).
```

Comment:
[ask] This bumping means a minor update, right?

Reply:
I think it's a minor update.
vscode@v1.82 was released half a year ago.
I think users who have disabled VSCode's automatic updates should also use older extensions.

Reply:
I see. Thanks for the answer!